test/Shared.ps1

```diff
@@ -1,4 +1,9 @@
 $modulePath = Convert-Path $PSScriptRoot\..\src
+# Test against built folder in CI (or just set $env:CI locally for that)
+if ($env:CI) {
```

Comment:
This works on a build machine. Can we also add a PowerShell variable check here like:
``
if ($env:CI -or $PoshGitBuild)
```
That way if we build & test from the build.ps1 script on a dev machine, it will run the Pester tests against the release dir.  And because that $PoshGitBuild variable goes away after the build script exits, I can still run tests from the src folder module when I doing dev work.